src/NuGet.Core/NuGet.Packaging/NuspecReader.cs

```diff
@@ -432,7 +432,7 @@ public LicenseMetadata GetLicenseMetadata()
 
                 var isKnownType = Enum.TryParse(type, ignoreCase: true, result: out LicenseType licenseType);
 
-                IList<string> errors = null;
+                List<string> errors = null;
```

Comment:
why change this from `IList` to `List`?

Reply:
I changed the warnings and errors to IReadOnlyList from IList. 

And List implements both :)